React/README.md

```diff
@@ -1,16 +1,17 @@
 # React
 
-1. [Intro to React](intro-to-react.md)
-1. [Hello World](react-hello-world.md)
+1.  [Intro to React](intro-to-react.md)
+1.  [Hello World](react-hello-world.md)
     - [Exercise](exercises/intro-to-react.md)
-1. [Creating Components](creating-components.md)
+1.  [Creating Components](creating-components.md)
     - [Worksheet: Building Basic and Nested Components](exercises/components-worksheet.md)
-1. [`props`](props.md)
-1. [Styling](styling.md)
-1. [`state`](state.md)
-1. [Event Handling](events.md)
-1. [Creating Components Part 2](creating-components-2.md)
-1. [Forms](forms.md)
-    - [Exercise: controlled forms](exercises/forms-worksheet.md)
-1. [Styling](styling.md)
-1. [Prop-Types](prop-types.md)
+1.  [`props`](props.md)
+1.  [Styling](styling.md)
+1.  [`state`](state.md)
+1.  [Event Handling](events.md)
+1.  [Creating Components Part 2](creating-components-2.md)
+1.  [Forms](forms.md)
+  - [Exercise: controlled forms](exercises/forms-worksheet.md)
+1.  [Styling](styling.md)
+1.  [Prop-Types](prop-types.md)
+1.  [Testing](testing.md)
```

Comment:
Should we just review this resource and not the "react-testing.md" one?

Reply:
Fixed, apologies